### DISCUSSION TITLE
Automatically save saved posts/comments into Unddit, Reveddit, Resavr AND save them locally

### DISCUSSION THREAD

**Discussion by ItsIgnacioPortal** [Docile-Alligator/Infinity-For-Reddit#1425] [Ideas]
I think it would be a good feature to have a setting that when enabled, would make it so when you save a comment/post, the post then gets archived by reveddit, resavr, etc. But also gets saved locally into the app, so if the comment/post gets deleted, whoever saved it can still view the archived version

**Comment by edgan** [comment]
`Continuum` now has the ability to save them locally when they fail to save on `Reddit`. This is mainly for `Removed` posts and comments.
